new-lamassu-admin/src/pages/Transactions/DetailsCard.js

```diff
@@ -0,0 +1,209 @@
+                            <div>{customer.country}</div>
+                          </div>
+                        </div>
+                        <div>
+                          <div>
+                            <Label>ID number</Label>
+                            <div>{customer.idCardNumber}</div>
+                          </div>
+                          <div>
+                            <Label>Gender</Label>
+                            <div />
+                          </div>
+                          <div>
+                            <Label>Expiration date</Label>
+                            <div>{customer.idCardExpirationDate}</div>
+                          </div>
+                        </div>
+                      </div>
+                    </IDButton>
+                  )}
+                  {tx.customerIdCameraPath && (
+                    <IDButton
+                      name='cam'
+                      Icon={CamIdIcon}
+                      InverseIcon={CamIdInverseIcon}
+                    >
+                      <img src={tx.customerIdCameraPath} />
+                    </IDButton>
+                  )}
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className={classnames(classes.col, classes.col2)}> {/* Column 2 */}
+            <div className={classes.innerRow}>
+              <div>
+                <Label>Exchange rate</Label>
+                <div>{`1 ${tx.cryptoCode} = ${Number(fiat / crypto).toFixed(3)} ${tx.fiatCode}`}</div>
+              </div>
+              <div className={classes.commissionWrapper}>
+                <Label>Commission</Label>
+                <div>{`${commission} ${tx.fiatCode} (${commissionPercentage * 100} %)`}</div>
+              </div>
+            </div>
+          </div>
+          <div className={classnames(classes.col, classes.col3)}> {/* Column 3 */}
+            <div className={classnames(classes.innerRow)}>
+              <div style={{ height: 43.4 }}>
+                {/* Export to PDF */}
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className={classnames(classes.row)}>
+          <div className={classnames(classes.col, classes.col1)}> {/* Column 1 */}
+            <div className={classes.innerRow}>
+              <div>
+                <Label>BTC address</Label>
+                <div>
+                  <Address type='btc'>
+                    {addr &&
+                      `${addr.slice(0, 5)}
+                      ${addr.slice(5, 10)}
+                      ${addr.slice(10, 15)}
+                      ${addr.slice(15, 20)}
+                      ${addr.slice(20, 25)}
+                      ${addr.slice(25, 30)}
+                      ${addr.slice(30, 35)}
+                      ${addr.slice(35, 40)}
+                      ${addr.slice(40, 45)}
+                      ${addr.slice(45, 50)}`}
+                  </Address>
```

Comment:
Shouldn't the address component handle the output? Also this won't work for every address, check out the `formatAddress` function on `Funding.js`